avm/res/net-app/net-app-account/capacity-pool/volume/main.bicep

```diff
@@ -181,7 +259,41 @@ resource volume 'Microsoft.NetApp/netAppAccounts/capacityPools/volumes@2023-07-0
   zones: zones
 }
 
-resource backupPolicies 'Microsoft.NetApp/netAppAccounts/backupPolicies@2023-11-01' = if (backupEnabled) {
+resource snapshotPolicies 'Microsoft.NetApp/netAppAccounts/snapshotPolicies@2024-03-01' = if (snapEnabled) {
```

Comment:
Interesting. This one (and the one below) 'should' be their own child-module of 'netAppAccounts', that would ultimately be published as its own module. This would require you to move all parameters specific to this resource into that nested child-module

Reply:
Hey @AlexanderSehr  understand however under some time constraints before my vacation tomorrow is it OK to gain approval on this PR now and I can come back to this and do once I return? Thanks Shaun

Reply:
@fbinotto  - Hi green badge now attached.  As above if this PR can be approved as is and once back from vacation I'll look to put the backup policies and snapshot policies into their own file...  unless someone else has some time to do while I'm OOTO ;)